client/src/components/about1/AboutPage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Content from "./Content";
+import { Link } from "react-scroll";
+
+function AboutPage() {
+  return (
+    <div>
+      <div 
+        className="hero min-h-screen"
+        style={{
+          backgroundImage:
+            "url(https://daisyui.com/images/stock/photo-1507358522600-9f71e620c44e.jpg)",
+        }}
+      >
+        <div className="hero-overlay bg-opacity-80"></div>
+        <div className="hero-content text-center text-neutral-content ">
+          <div className="max-w-lx">
+            <h1 className="text-[45px] font-medium  text-sky-50">
+              Welcome to <span className="underline font-bold">DEJA</span> Review 
+            </h1>
+            <h1 className="mb-6 text-[32px] font-medium underline  text-sky-50">
+              Spaced Repetition technique
+            </h1>
+
+            <div className="text-2xl mb-6 text-sky-50 font-semibold">
```

Comment:
It was a cool idea to add the motto to the top of this page. But it looks too static. 
Could you consider implementing a feature that would animate this text like:
**Where you always**(text stay static) and other part is animated to change every 2ish second